pages/write/index.tsx

```diff
@@ -39,7 +39,7 @@ export default function WritePage() {
                     <CollectionCard
                         collectionKey={'kanjigarden'}
                         title={'KANJI GARDEN'}
-                        description={'Kanji Garden is a free mnemonic-based SRS kanji learning tool that features about 2900 kanji. '}
+                        description={'Kanji Garden is a free mnemonic-based SRS kanji learning tool that features about 2600 kanji. '}
```

Comment:
After removing all the unnecessary radicals and katakana from the list, 2830 kanji remain

Reply:
If you see the latest source file in `generator/sources/kanjigarden/all.source`, there are exactly 2645 kanjis. (I removed extra radicals)

Reply:
Ah, okay then!